README.md

```diff
@@ -28,29 +28,61 @@ See example [`Dockerfile`](hype-docker/Dockerfile)
 ## Submit
 
 Create a hype `Submitter` and use `runOnCluster(fn, env)` to run a closure in your docker container
-environment on a Kubernetes cluster. This example runs a simple block of code that just constructs
-a list of all environment variables and returns it.
+environment on a Kubernetes cluster. This example runs a simple block of code that just extracts
+the operating system name and lists all environment variables.
 
 ```scala
-// A function to run in a docker container
-val fn: Fn[List[String]] = () => for {
-  (key,value) <- sys.env.toList
-} yield s"$key=$value"
+import com.spotify.hype._
+import scala.sys.process._
+
+// A simple model for describing the runtime environment
+case class EnvVar(name: String, value: String)
+case class Res(uname: String, vars: List[EnvVar])
+
+def extractEnv: Res = {
+  val uname = "uname -a" !!
+  val vars = for ((name, value) <- sys.env.toList)
+      yield EnvVar(name, value)
+
+  Res(uname, vars)
+}
 
 // Use a Google Cloud Container Engine managed cluster
-val cluster = ContainerEngineCluster.containerEngineCluster(
+val cluster = containerEngineCluster(
     "gcp-project-id", "gce-zone-id", "gke-cluster-id") // modify these
 
-val env = RunEnvironment.environment(
-    "gcr.io/gcp-project-id/env-image", // the env image we created earlier
-    Secret.secret("gcp-key", "/etc/gcloud")) // a pre-created k8s secret volume named "gcp-key"
+val env = environment("gcr.io/gcp-project-id/env-image") // the env image we created earlier
```

Comment:
Question about the image provided above. Is it not still required to have the GOOGLE_APPLICATION_CREDENTIALS set in the docker image? Last time I tested without it, the staged files could not be accessed when running on kubernetes.

This example has us setting a secret for the key without using it anywhere, I think it would make more sense to show why we are adding that secret.

Also I still am for providing a base image in the public registry with the most basic bare bones required to run a hype task to allow users to try it with ease. (especially now that we have the local runner)